test/test_logging.py

```diff
@@ -24,3 +26,4 @@ def test_import_tabular():
         os.rmdir(not_existing_dir)
     except PermissionError:
         pass
+    os.environ.pop(INTERPRET_C_LOGS)
```

Comment:
In the future it might be good to monkeypatch this in case the runner of the test is a dev that wanted to set the INTERPRET_C_LOGS env var.

Reply:
"in case the runner of the test is a dev that wanted to set the INTERPRET_C_LOGS env var"
they could just set that env variable directly

Reply:
At the end of the test you pop the variable which means after running the test the state of the system would be different if the dev had set the variable before running the tests

Reply:
Pretty sure it doesn't fully meet testing idempotence requirements